bash_completion

```diff
@@ -245,6 +245,53 @@ _upvars()
     done
 }
 
+# Get the list of filenames that match with the specified glob pattern.
```

Comment:
I think a little reminder here why we should be using this this would be a nice touch.

```suggestion
# Get the list of filenames that match with the specified glob pattern.
# This function does the globbing in a controlled environment, avoiding
# interference from user's shell options/settings or environment variables.
```

Reply:
Thank you! I have applied it 62e2440f.